src/deluge/playback/playback_handler.cpp

```diff
@@ -2506,6 +2511,27 @@ bool PlaybackHandler::tryGlobalMIDICommands(MIDIDevice* device, int channel, int
 	return foundAnything;
 }
 
+// Returns whether the message has been used up by a note-off command
+bool PlaybackHandler::tryGlobalMIDICommandsOff(MIDIDevice* device, int channel, int note) {
+
+	bool foundAnything = false;
+
+	for (int c = 0; c < kNumGlobalMIDICommands; c++) {
+		if (midiEngine.globalMIDICommands[c].equalsNoteOrCC(device, channel, note)) {
+			switch (static_cast<GlobalMIDICommand>(c)) {
+			case GlobalMIDICommand::FILL:
+				currentSong->fillModeActive = false;
+				indicator_leds::setLedState(IndicatorLED::SYNC_SCALING, false);
```

Comment:
Same here as my comment above

Reply:
Yep, same. It's leftover from the initial implementation. Next task is to add the community setting.